run-queue

```diff
@@ -64,7 +64,7 @@ def consume_webhook_queue(channel, amqp):
         elif action == 'closed' and pull_request.get('merged', False):
             sha = pull_request['head']['sha']
             db = os.path.join(get_images_data_dir(), "test-results.db")
-            cmd = "./store-failures --db {0} --repo {1} {2}".format(db, repo, sha)
+            cmd = "./store-tests --db {0} --repo {1} {2}".format(db, repo, sha)
```

Comment:
This could be squashed into the previous commit -- that is already rather inconsistent as it mixes "-failures" with "tests.db" and such.